ubuntu/provision/libvmi.yml

```diff
@@ -24,16 +25,16 @@
     dest: /home/vagrant/libvmi
   become: false
 
-- name: generate configure
-  command: ./autogen.sh
+- name: creates build directory
+  command: mkdir build
```

Comment:
if you want to be full Ansibile, use the file module:
~~~YAML
name: create build directory
file:
  path: /home/vagrant/libvmi/build
  state: directory
~~~

something like that, check the file module